go-controller/pkg/ovn/policy.go

```diff
@@ -325,10 +335,11 @@ func (oc *Controller) createDefaultDenyPortGroup(ns string, nsInfo *namespaceInf
 }
 
 // modify ACL logging
-func (oc *Controller) setACLDenyLogging(ns string, nsInfo *namespaceInfo, aclLogging string) error {
+func (oc *Controller) setDefaultACLsLogging(ns string, nsInfo *namespaceInfo, aclLogging string) error {
```

Comment:
This function needs an eventual rethink -- we shouldn't be reliant on the exact condition of the match; we should be setting ACLs with discoverable external_id's. But that's for another day...

Reply:
I totally agree, I really dislike querying acl by match... External ID, name, or UUID would all be superior in my mind